doc/index.html

```diff
@@ -4,7 +4,7 @@
 <title>Page Title</title>
 <meta charset="UTF-8">
 <meta name="viewport" content="width=device-width, initial-scale=1">
-<meta http-equiv = "refresh" content = "3; url = https://sig-product-docs.synopsys.com/bundle/integrations-detect/page/introduction.html" />
+<meta http-equiv = "refresh" content = "3; url = https://documentation.blackduck.com/bundle/detect/page/introduction.html" />
```

Comment:
@sig-cpotts @andrian-sevastyanov Should this be https://documentation.blackduck.com/bundle/integrations-detect/page/introduction.html instead, I see 404 for the current link?

Reply:
This would be https://documentation.blackduck.com/bundle/integrations-detect/page/introduction.html right up until we release 10.0.0 and it becomes https://documentation.blackduck.com/bundle/detect/page/introduction.html
BUT I may see if I can manually upload the 9.10.0 docs under the  https://documentation.blackduck.com/bundle/detect path before that, so they are available at both locations while we are migrating. So if we aren't pushing this right away, I'll see about getting that done first so that the URL Andrian has, can remain.

Reply:
Thanks @devmehtasynopsys and @sig-cpotts.

Sounds like having documentation temporarily available under both `detect` and `integrations-detect` is preferable to me changing this PR to point at `integrations-detect` because it will help us avoid getting into a situation where the link breaks again when Detect 10 is released.

So, I'll hold off from merging these changes until we have docs available under both bundles.

Reply:
@andrian-sevastyanov Posted 9.10.0 docs to the new URL in addition to the old one, so we are good to go.
https://documentation.blackduck.com/bundle/detect/page/introduction.html

Reply:
Great, thank you!